src/Clock.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+import React, { useState, useEffect } from 'react';
+
+type Props = {
+  name: string;
+};
+
+export const Clock: React.FC<Props> = ({ name }) => {
+  const [date, setDate] = useState(new Date());
+
+  useEffect(() => {
+    const timerId = window.setInterval(() => {
+      setDate(new Date());
+      console.info(date.toUTCString().slice(-12, -4));
+    }, 1000);
+
+    return () => {
+      window.clearInterval(timerId);
+    };
+  }, []);
+
+  useEffect(() => {
+    console.debug(`Renamed from ${name} to ${name}`);
+  }, [name]);
```

Comment:
You only need the current name so you can remove `from ${name}` but also add a condition so it doesn't print the first name